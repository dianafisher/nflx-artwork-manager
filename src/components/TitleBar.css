.titlebar {
  flex-shrink: 0;
  width: 100%;
  position: relative;
  background-color: #df2230;
  height: 50px;
  color: #fff;
  margin-bottom: 40px;
  box-shadow: 0 3px 3px rgba(0,0,0,.29);
  z-index: 2;
}

.titlebar-title {  
  height: 48px;
  font-size: 20px;
  color: #fff;
  display: flex;
  align-items: center;
  padding-left: 20px;
}
