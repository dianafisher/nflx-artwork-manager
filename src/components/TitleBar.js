import React from 'react';
import './TitleBar.css';

function TitleBar (props) {

  return (
    <div className='titlebar'>
      <div className='titlebar-title'>Netflix Artwork Manager</div>
    </div>
  )
}


export default TitleBar;
