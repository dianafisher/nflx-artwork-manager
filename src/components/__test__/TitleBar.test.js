import React from 'react';
import { shallow } from 'enzyme';
import TitleBar from '../TitleBar';

describe('TitleBar', () => {

  it('renders', () => {
    expect(shallow(
      <TitleBar/>
    )).toMatchSnapshot();
  });
});
