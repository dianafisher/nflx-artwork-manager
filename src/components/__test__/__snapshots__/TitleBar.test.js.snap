// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleBar renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="titlebar"
>
    <div
        className="titlebar-title"
    >
        Netflix Artwork Manager
    </div>
</div>,
  "nodes": Array [
    <div
      className="titlebar"
>
      <div
            className="titlebar-title"
      >
            Netflix Artwork Manager
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <TitleBar />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="titlebar"
>
          <div
                    className="titlebar-title"
          >
                    Netflix Artwork Manager
          </div>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="titlebar"
>
          <div
                    className="titlebar-title"
          >
                    Netflix Artwork Manager
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <TitleBar />,
}
`;
